drivers/saul/init_devs/auto_init_hsc.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2020 Deutsches Zentrum für Luft- und Raumfahrt e.V. (DLR)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_auto_init_saul
+ * @{
+ *
+ * @file
+ * @brief       Auto initialization of HSC driver.
+ *
+ * @author      Quang Pham <phhr_quang@live.com>
+ *
+ * @}
+ */
+
+#ifdef MODULE_HSC
+
+#include "assert.h"
+#include "log.h"
+#include "saul_reg.h"
+#include "hsc.h"
+#include "hsc_params.h"
+
+#define LOG_LEVEL LOG_DEBUG
+#define ENABLE_DEBUG 1
```

Comment:
```suggestion
```

By default, debug output should be disabled. As `debug.h` is not included, no `ENABLE_DEBUG` define is actually needed.